components/data/librarysize/__init__.py

```diff
@@ -0,0 +1,125 @@
+#loadlibraries
+from scipy import datasets
+from sklearn.decomposition import PCA
+import plotly.graph_objs as go
+from plotly.offline import iplot, plot_mpl
+import scipy.stats as ss
+import warnings, sys, os
+import pandas as pd
+from IPython.display import display, Markdown
+import chart_studio.plotly as py
+import json
+from components.data.gene_count_matrix import anndata_from_file
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import FunctionTransformer
+import numpy as np
+import pandas as pd
+import chart_studio.plotly as py
+import sys, os
+from matplotlib import figure
+import plotly.graph_objs as go
+from plotly.offline import iplot
+from IPython.display import display, Markdown
+
+##### 2. Other libraries #####
+sys.path.append(os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__)))), 'core_scripts', 'shared', 'shared.py'))
+import shared as s
```

Comment:
This file doesn't exist anywhere, ideally we should try and make it not necessary, but if it is, it needs to be added somewhere in this project